static/js/pages/independent-expenditures.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+/* global require, document */
+
+var $ = require('jquery');
+
+var tables = require('../modules/tables');
+var helpers = require('../modules/helpers');
+var columns = require('../modules/columns');
+var FilterPanel = require('../modules/filter-panel').FilterPanel;
+
+var expenditureTemplate = require('../../templates/independent-expenditures.hbs');
+
+var columns = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'min-desktop',
+    render: function(data, type, row, meta) {
```

Comment:
'meta' is defined but never used.<br>'row' is defined but never used.<br>'type' is defined but never used.